package.json

```diff
@@ -13,8 +13,8 @@
   ],
   "main": "index.js",
   "scripts": {
-    "test": "xo && mocha test/**/*.js",
-    "lint": "xo"
+    "lint": "xo",
+    "test": "nyc mocha test/**/*.js"
```

Comment:
Ok I see now, `nyc` is a code coverage reporter. But why separate our lint checking? Best practice is to lint (`xo`) and run tests and then run `test` to get both, all the time.

Reply:
I can change it back...just seemed like test doing a bit too much to me.

Reply:
If there is no functional regression to the `nyc mocha...` stuff then ideally test would include the linter. A failure to adhere to our style guide should fail a PR, and it is preferable to put it into `test` so developers don't have to run two commands to know their PR will pass checks.